misc/config_tools/schema/checks/passthrough_devices.xsd

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<xs:schema xml:id="root"
+	   xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	   xmlns:acrn="https://projectacrn.org">
+
+  <xs:assert test="every $vm in /acrn-config/vm satisfies
+                   starts-with($vm//vm_type, 'PRE') or not($vm//mmio_resources/p2sb = 'y')">
+    <xs:annotation acrn:severity="error">
+      <xs:documentation>VM {$vm/@id} is not a pre-launched VM and cannot have a P2SB (Primary-to-Sideband bridge) passed through to it.  Remove the mmio_resources.p2sb value for this VM.</xs:documentation>
```

Comment:
Does this message appear in the context of the ACRN Configurator UI? If so, consider writing the action in that context. For example, instead of "Remove the mmio_resources.p2sb value for this VM" say "Clear this option" (assuming it's a checkbox in the UI). Same comment for all messages.

Reply:
These documentation can appear in three places:

1. As error messages in the ACRN configurator.
2. As build-time errors in terminal.
3. In web pages on https://projectacrn.github.io/.

Only in the first case will an error be related to one (or more) widget.

Reply:
I can't tell which messages will be in the UI, but let's consider whether an additional UI message field is needed for those messages.

Reply:
Let's revisit this once the UI has integrated the validation rules and shown validation errors. Based on what we have at that time, we can see if any message about additional actions is needed.

Reply:
Sounds good, thanks